EIPS/eip-615.md

```diff
@@ -290,7 +291,7 @@ All of the remaining conditions we validate statically.
 
 #### Costs & Codes
 
-All of the instructions are `O(1)` with a small constant, requiring just a few machine operations each, whereas a `JUMP` or `JUMPI` must do an `O(log n)` binary search of an array of `JUMPDEST` offsets before every jump. With the cost of `JUMPI` being _high_ and the cost of `JUMP` being _mid_, we suggest the cost of `JUMPV` and `JUMPSUBV` should be _mid_, `JUMPSUB` and `JUMPIF` should be _low_, and`JUMPTO` and the rest should be _verylow_.  Measurement will tell.
```

Comment:
Why remove the formatting here? Isn't it better to keep it consistent with the previous one?

Reply:
No reason, an accident.  Also, Martin informs me, false.  He uses a bitmap to check jump destinations in O(1).